scaladoc/src/dotty/tools/scaladoc/tasty/ClassLikeSupport.scala

```diff
@@ -79,6 +79,7 @@ trait ClassLikeSupport:
         else Seq(link -> superLink) ++ getSupertypesGraph(superLink, nextTo)
       }
 
+    // TODO use classDef.baseTypes
```

Comment:
@pikinier20 SInce you created this PR the code below changed. Is this method still relevant or is `Symbol.baseCalsses` enough?